Aztec/Classes/Plugin/Plugin.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+open class Plugin {
```

Comment:
Will be good to add documentation to the class and methods to explain what is a plugin and how to use, and where they act on the Aztec parsing process.